core/types/calls.cc

```diff
@@ -876,11 +913,12 @@ DispatchResult dispatchCallSymbol(const GlobalState &gs, const DispatchArgs &arg
     // If positional arguments remain, the method accepts keyword arguments, and no keyword arguments were provided in
     // the send, assume that the last argument is an implicit keyword args hash.
     bool implicitKwsplat = false;
+    UnorderedSet<Loc> kwErrors;
```

Comment:
This is also a hack, unfortunately.

Currently there is two places where the implicit conversion generates errors:
1. https://github.com/sorbet/sorbet/blob/f0fa23117cbdb067b20e2cdd3ed588d7142b0906/core/types/calls.cc#L921-L931
2. https://github.com/sorbet/sorbet/blob/f0fa23117cbdb067b20e2cdd3ed588d7142b0906/core/types/calls.cc#L1087-L1101

Because of that some cases are reported twice. I didn't manage to figure out how to make them play well with each other, so I decided to dedup errors